### ISSUE TITLE
Download on safari

### ISSUE THREAD

**Issue by yoan1005** [rap2hpoutre/fast-excel#41]
Hi,

I use it for download an export of table basically. 
It work perfectly on google chrome but on safari, he had .html after my .xlsx 

have you an idea ?
Thanks

**Comment by elminson** [comment]
Please, @yoan1005 provide the version of Safari and the OS specifications! I can take a look but I need this information.

**Comment by elminson** [comment]
I just try with and not html in the file was found 
Safari 5.1.7 For Windows (I don't have a Mac) 
![fast-excel screenshot_3](https://user-images.githubusercontent.com/2476286/43271533-c4396e8a-90c5-11e8-982c-3b5e4ab47376.png)
![fast-excel - screenshot_38](https://user-images.githubusercontent.com/2476286/43271534-c44dfe9a-90c5-11e8-8922-73ab14b3a001.png)

**Comment by yoan1005** [comment]
Hi 

Thanks for anwser

my specs : 
<img width="337" alt="capture d ecran 2018-07-26 a 17 59 48" src="https://user-images.githubusercontent.com/14938326/43273877-f3c7b9f8-90fd-11e8-90c2-dbc64c9ef383.png">
<img width="632" alt="capture d ecran 2018-07-26 a 17 59 55" src="https://user-images.githubusercontent.com/14938326/43273878-f3e68ea0-90fd-11e8-8a88-f04d2d828f88.png">

**Comment by elminson** [comment]
Thanks, @rap2hpoutre can you please debug this? I don't have a Mac for a test. 

Elminson

**Comment by rap2hpoutre** [comment]
OK I have the same problem, sorry for the delay. 

@yoan1005 Do you use the `download` method or the `export` method? If you use the `download` method, could you try to use the `export` method then use the standard laravel method for downloading? (https://laravel.com/docs/5.6/responses#file-downloads)

Example:

```php
public function myActionInAController() {
    (new FastExcel(User::all()))->export('file.xlsx');
    return response()->download('file.xlsx');
}
```

**Comment by rap2hpoutre** [comment]
@yoan1005 A fix in v0.8.0 about download _may_ have fixed your issue. You could also just try to update to this version
